tests/_test_mscolab/test_file_manager.py

```diff
@@ -74,12 +74,12 @@ def test_modify_user(self):
             # cannot create a user a second time
             assert self.fm.modify_user(user, action="create") is False
             # confirming the user
-            confirm_time = datetime.datetime.now() + datetime.timedelta(days=1)
+            confirm_time = datetime.datetime.now(tz=datetime.timezone.utc) + datetime.timedelta(days=1)
             self.fm.modify_user(user_query, attribute="confirmed_on", value=confirm_time)
             self.fm.modify_user(user_query, attribute="confirmed", value=True)
             user_query = User.query.filter_by(id=user.id).first()
             assert user_query.confirmed is True
-            assert user_query.confirmed_on == confirm_time
+            assert user_query.confirmed_on.replace(tzinfo=None) == confirm_time.replace(tzinfo=None)
```

Comment:
I think this change hides a bug.

When I revert this I get the following error:
```
FAILED tests/_test_mscolab/test_file_manager.py::Test_FileManager::test_modify_user - assert datetime.datetime(2024, 2, 9, 8, 49, 39, 968400) == datetime.datetime(2024, 2, 9, 8, 49, 39, 968400, tzinfo=datetime.timezone.utc)
```

This means that the database query returns a naive datetime object, which is then not equal to the timezone aware object. I think the database query should return an aware datetime object instead, but I am not yet sure how to make it do that.

Reply:
Ok I didn't realise this situation. I will look into it.

Reply:
This can also be handled in a follow-up PR because it is not exactly related to the initial issue that is addressed here. But I would prefer if this line stays as is (i.e. `assert user_query.confirmed_on == confirm_time`), and the test be marked as xfail instead to acknowledge that this is an issue.